contracts/0.8.9/CommitteeQuorum.sol

```diff
@@ -0,0 +1,219 @@
+// SPDX-FileCopyrightText: 2022 Lido <info@lido.fi>
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.9;
+
+import "./lib/AragonUnstructuredStorage.sol";
+
+
+/**
+ * @title Implementation of oracle committee consensus reporting
+ */
+contract CommitteeQuorum {
+    using UnstructuredStorage for bytes32;
+
+    event MemberAdded(address indexed member);
+    event MemberRemoved(address indexed member);
+    event QuorumChanged(uint256 quorum);
+
+    /// Maximum number of oracle committee members
+    uint256 public constant MAX_MEMBERS = 256;
+
+    /// Number of exactly the same reports needed to finalize the epoch
+    bytes32 internal constant QUORUM_POSITION = keccak256("lido.CommitteeQuorum.quorum");
+
+    uint256 internal constant MEMBER_NOT_FOUND = type(uint256).max;
+
+    /// The bitmask of the oracle members that pushed their reports
+    bytes32 internal constant REPORTS_BITMASK_POSITION = keccak256("lido.CommitteeQuorum.reportsBitmask");
+
+    ///! STRUCTURED STORAGE OF THE CONTRACT
```

Comment:
maybe go with unstructured since we use a contemporary solidity version
https://gist.github.com/skozin/344d9c7d2270cac0c3f9373c2f43c846